ui/apps/platform/src/Containers/Integrations/IntegrationTiles/IntegrationTilesPage.tsx

```diff
@@ -25,6 +26,7 @@ function IntegrationTilesPage(): ReactElement {
                 <SignatureIntegrationsSection />
                 <NotifierIntegrationsSection />
                 {canUseCloudBackupIntegrations && <BackupIntegrationsSection />}
+                <CloudSourceIntegrationsSection />
```

Comment:
FYI in theory this would have conditional rendering for ROX_CLOUD_SOURCES feature flag like `{isCloudSourcesEnabled && <CloudSourceIntegrationSection />}` but in reality we are close to turn it on for release.

Reply:
I wouldn't want to put it under the feature flag unless we create a ticket to switch feature flag + TODO here